hs_communities/management/commands/hide_czo_national.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group
+
+
+def hide_national_group():
+    """
+    CZO National is used to establish Community membership and privileges. It should be hidden from user view.
+
+    :return:
+    """
+    national_group = list(Group.objects.filter(name="CZO National"))
```

Comment:
This command seems too specific to CZO and I'm not sure what it's even for.  If you need to make the CZO group unlisted in a management command, it may make more sense to write a management command that can make any group unlisted and pass the name of the group to the command.

Reply:
I believe this is changing, per conversation at all hands. Waiting for @alvacouch to return

Reply:
... as in, this was discussed and I anticipate it to change in a way that would likely address your sense here.